test/e2e/common/security_context.go

```diff
@@ -256,13 +256,13 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		})
 
 		/*
-		  Testname: allowPrivilegeEscalation=true.
-		  Description: Configuring the allowPrivilegeEscalation to true, allows the privilege escalation operation.
-		  A container is configured with allowPrivilegeEscalation=true and a given uid (1000) which is not 0.
-		  When the container is run, the container is run using uid=0 (making use of the privilege escalation).
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID.
+			Release : v1.14
+			Testname: Security Context, allowPrivilegeEscalation=true.
+			Description: Configuring the allowPrivilegeEscalation to true, allows the privilege escalation operation.
+			A container is configured with allowPrivilegeEscalation=true and a given uid (1000) which is not 0.
+			When the container is run, container's output MUST match with expected output verifying container ran with uid=0 (making use of the privilege escalation).
 		*/
-		It("should allow privilege escalation when true [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should allow privilege escalation when true [NodeConformance]", func() {
```

Comment:
I don't think this is part of conformance.  A container runtime may choose to not allow this.

Reply:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.

Reply:
Not Promoting this e2e to Conformance. Noted reason in the code comment.